config/src/main/java/org/springframework/security/config/annotation/web/reactive/ReactiveOAuth2ClientImportSelector.java

```diff
@@ -64,14 +66,12 @@
 
 		private ReactiveOAuth2AuthorizedClientService authorizedClientService;
 
+		private ReactiveOAuth2AuthorizedClientManager authorizedClientManager;
```

Comment:
I don't think this belongs in this PR. Can you please double-check your rebase steps?